.changeset/mighty-eagles-wink.md

```diff
@@ -0,0 +1,8 @@
+---
+"@llamaindex/ollama": minor
+"@llamaindex/openai": minor
+"@llamaindex/core": minor
+"@llamaindex/examples": minor
+---
+
+Added support for structured output in the chat api
```

Comment:
need to update https://ts.llamaindex.ai/docs/llamaindex/tutorials/structured_data_extraction